### ISSUE TITLE
Rules rework

### ISSUE THREAD

**Issue by ecoerod** [progdisc/rules#13]
It's that time of the year again, where we look at our rules and wonder if they are actually suitable for our server, if they convey the information we want to convey while at the same time avoiding being overwhelming or scary.

In order to keep the information contained and easy to follow, we will track the suggestions and proposals for change in this issue.

As of currently, while everything is subject to change, we want to keep the actual content of the rules mostly intact, as we consider that it still reflects the philosophy and values that the server wants to maintain. We are open to reasonable changes or additions in this regard, though, as always.

**Comment by CameronAavik** [comment]
In my view, the reason we have the rules posted in the #rules channel is to increase the likelihood that people will read them as they may not want to go to an external website to view them. While this is still a good idea and I believe has helped reduce accidental rule violations, there are a number of sections in the rules which new members don't necessarily need to know about before participating as they only cater to perhaps the 5% of users who are interested in them, and they are just informational. 

Therefore I suggest that the "Forms of Punishment", "Helpers", and "Mods" sections of the rules be removed from the #rules channel and instead are hosted on our website with a link to there in #rules for people who are interested to read that information.

The size of the #rules channel is quite daunting and so containing it to just the things we need people to read before participating will help make it less daunting.

**Comment by CameronAavik** [comment]
Another suggestion:
Rule 8 is actually three rules and so some people might only read the first part without reading the rest thinking it is just an elaboration on the rules. We should split them up into three rules.

>8. Do not post or discuss NSFW/NSFL or illegal content (this includes nicknames). Do not post or discuss piracy content. Do not post or discuss political content.

**Comment by saethlin** [comment]
I agree that the rules are too long. Maybe we should also look to shorten up wording for the main set; the fact that "No unsolicited DMs" falls into the second embed may be to blame for the number of errant DMs we get. It would be nice if everything fit in one.

I would really like a consistent numbering system or such that lets us quickly refer people to the rules. If someone says "rule 3" there are five items that are numbered as 3 in rules channel, and three of them are rules. Breaking off the sections for mods, helpers, and forms of punishment would help clarify this though there's still the second section for #security.

**Comment by Samoxive** [comment]
Please also keep in mind that embeds have a 1024 characters limit for their description field. Splitting the document to only contain the rules sounds like a good idea.

**Comment by spectras** [comment]
Description should be 2048 I believe. It's individual fields that are limited to 1024.
Anyway, the shorter the text, the higher the chance it will be read.

**Comment by article10echr** [comment]
I do not want to make a new issue for this and this is where the rules discussion is going on, so here goes.

Rule 8 states: 'Do not post or discuss political content.'

I've been subscribed to the Discord for months. Now, minutes ago I was preparing a message in the Discord's general chat to ask about moving to the intermediate level when I saw multiple terms which in most conversation would signal that it is about politics. More specifically there was some discussion going on that relates to 'identity politics' in particular in relation to social identity (the academic definition of identity politics contains 'asserting group distinctiveness and belonging and gaining power and recognition'). I didn't read the entire discussion because I considered the wording prima facie evidence (I did look up the definition of identiry politics later and turned out to be correct) that there was at least some degree of politics which I felt was out of place in a programming Discord, please note that this was distracting me while I was trying to compose aforementioned message. Someone's social identity should in principle not be brought up either in affirmation or disparagement.

So I paused composing my message and just said: "oh no, not this community too, can we please keep politics out of here? Thank you". In my view my request was fully within rule 8, merely repeating it almost, and also appropriate given the subject of the Discord, programming, not social politics.

Then within seconds, not even being warned that what I said would violate any rule (I'm sure I wasn't being uncivil), I was permanently banned and... called an asshole (!) by admin Mike Overby, lethargillistic#2728 (who is also a participant at this GitHub over at https://github.com/lethargilistic
)

Full ban message:

You were banned from Programming Discussions

Reason:

Pride is only politics if you're an asshole who doesn't deserve to be here.

Banned until:
Indefinitely 

Screenshot: https://i.imgur.com/zrmukRB.jpg

I would like to request an unban and an apology from Mike Overby for incivility as well as for 1. Incorrectly banning, 2. Banning without warning and 3. Bringing politics which don't belong here.

I couldn't reply to the ban notification in Discord and neither could I post in any other channel in the Discord since I was fully banned. 

Kind regards, and all the best with the Discord, regardless of your decision!

**Comment by lethargilistic** [comment]
You defended the behavior of an extremely transphobic individual. I do not give a single damn if you "civilly" approached defending "discussion" about the person who started talking about trans people in those terms and caused another individual to leave the server because of the inappropriate content.

We do not do that in this community.

You're out and you can stay out.

**Comment by lethargilistic** [comment]
Unbeknownst to me, the other moderators decided to unban you.

I explicitly do not apologize for banning you in the first place.

I do, however, apologize for sending the mixed message. If the others have welcomed you back, then you are welcome.

**Comment by article10echr** [comment]
Mike's assertion that I 'defended the behavior of an extrmely transphobic individual' is simply false. Here is a screenshot of the extent of my comments in the thread: https://i.imgur.com/vxWlej8.jpg

I just saw Mike used the words 'transphobic shit'  I didn't even see the previous commente because the user that posted them was also aparently banned and their comments removed before I even entered #general. I made my 'no politics' request and that was that: permaban.

Mike's ban of me also incited more violations of rule 8: Kenneth Love (pinging https://github.com/kennethlove ) started saying 'programming is political', which is stretching the definition of the adjective until it's meaningless but in any case violates rule 8 and goes against what rule 8 is trying to prevent. 

As someone said later: man the perm ban trigger seems really really loose.

**Comment by lethargilistic** [comment]
My name is Mike, which you know.

That screencap does not include the ban of the person before you, which you know.

You're already allowed back, which you know.

**Comment by ecoerod** [comment]
First: this is really not the place to debate this, you should've just gone to a mod directly in Discord (or even opening a new issue).

On the case of the ban: it has always been the policy of this server that inclusiveness is absolute and inalienable, Pride Month or not. The display of the flag on this month was meant as an encouraging symbol to a community that suffers terrible situations on a daily basis, especially in the technological fields, and we've gotten many, many messages of encouragement and appreciation of this gesture, that thoroughly trump any heat we've gotten over the same matter.

Second: on the "programming is always political". While the message sounds broad, it's entirely true, because our profession deals with solving problems for people, and thus, as long as we live in a society, politics is a part of it. On the other hand, the rule about "not discussing political content" refers more explicitly to content that could be found on an election season (memes and messages about candidates, direct political views and debates...), because these always devolve into flame wars and shit-flinging competitions. LGBT+ issues are not, or SHOULD NOT, be a matter of political debate, and that's the stance of this server as expressed in rule 1.

On the matter of posting political content, the expression of inclusiveness towards the LGBT+ community in this server limits itself to the display of the Pride Flag in the background of the icon for this month. There's no direct mention of it in the server, no debate or discussion over its merits (and we cut it as soon as it starts), or anything of the sort. The content of the server is still strictly programming, a different icon should not prompt this response, and that it did makes me personally question your motives about stirring this entire debate. 

The ban was issued due to your unfortunate timing. As we said, another user was driver out of this server by the user banned previously, and you were caught on the line of fire for seemingly challenging this action. 

You were only unbanned as there was minimal doubt in that your messages were in challenge or not of this moderation action against an unwelcome character in our community, nothing more.

This debate ends now.
